specification/automation/resource-manager/Microsoft.Automation/common/v5/definitions.json

```diff
@@ -0,0 +1,118 @@
+{
+  "swagger": "2.0",
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "info": {
+    "version": "2018-06-10",
+    "title": "Common types"
+  },
+  "paths": {},
+  "definitions": {
+    "PrivateEndpointConnection": {
```

Comment:
Pls check to refer these from common types. I see `privateEndpoint` related object types are similar to the ones available in the common-types here - `specification/common-types/resource-management/v5/privatelinks.json`